util/versionutil/feature_flag.go

```diff
@@ -37,8 +35,8 @@ func NewFeatureFlag(name string, constraints []string) *FeatureFlag {
 // constraints examples: "~5.2.2", ">= 5.3.0", see semver docs to get more information
 func (ff *FeatureFlag) IsSupported(targetVersion string) bool {
 	curVersion := targetVersion
-	if version.Standalone == "No" {
-		curVersion = version.PDVersion
+	if Standalone == "No" {
```

Comment:
Standalone is a concept of TiDB Dashboard instead of other variants. Can we make it more general so that we can use it in Debug Portal or other variants?